Week2/homework/js-exercises/CollectiveAge/collectiveAge.js

```diff
@@ -0,0 +1,17 @@
+const hackYourFutureMembers = [
+    { name: 'Wouter', age: 33 },
+    { name: 'Federico', age: 32 },
+    { name: 'Noer', age: 27 },
+    { name: 'Tjebbe', age: 22 },
+];
+
+const calculateSum = array => {
+    return array.reduce((sum, item) => sum + item, 0);
+}
```

Comment:
Great! Formatting note. In arrow functions you can return without using the return keyword if you omit the curly brackets. Like this:
`const calculateSum = array => array.reduce((sum, item) => sum + item, 0)`